### PR TITLE
Restaurant Page Alpha Release

### PR DESCRIPTION
**Pull request by HeflerDev**
This is the Alpha Release of the Restaurant Page Project.
I had an issue with git, precisely with the .gitignore file, which was blocking my dist folder and I just noticed after discarding the changes.
So I had to do all the distribution part again due to this problem (that's why in the last commits I pushed only the visuals).
Despite that, the project is now ready to review.

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Nice work on this project, but the following changes need to be done before moving on ⬇️** 

* [x] The app should append HTML elements to the `<div>` with `id=content` when the page is loaded (Use `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)) the HTML code in `index.html` in the `dist` directory should be removed except for the `<div>` with `id=content`.

* [x] The contents of each tab are stored in a `module`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).

* [x] Each `module` exports a function that creates a `<div>` element, adds the appropriate content and styles to that element and then appends it to the DOM. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).

* [x] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

* [x] Necessary run scripts should configured in the `package.json` file, like `build`, `development` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)).

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A project that counts towards Microverse graduation.",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack"
   },
   "repository": {
```
Comment (shloch) [inline package.json:10]: Hello great job with the ameliorations. However please we have one last thing (small detail) regarding the LAST comment of former reviewer. Please add these lines to your `package.json` file :
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

``

Then we'll be good to go

**Review by shloch — CHANGES_REQUESTED** [review]
## Great job

Please one more thing left..... see below

**Review by shloch — APPROVED** [review]
accomplished

### Good to go ....

**Pull request merged**